_posts/2021-02-23-OpenSSL.markdown

```diff
@@ -0,0 +1,58 @@
+---
+title: Switching back to OpenSSL
+layout: post
+---
+
+The Void Linux team is planning on [performing the switch to
+OpenSSL](https://github.com/void-linux/void-packages/pull/21056) before March
+is over.
+
+For most users, this shouldn't be noticeable, since we will be pushing the same
+packages that already had to be rebuilt twice a year due to LibreSSL ABI
+breaks, but if anything in your setup depends explicitly on LibreSSL (you know
+who you are!), be sure to migrate away from that.
+
+The important parts of the post are done, but you can read on to learn about
+the decision process and some things we had to do.
+
+The discussion started in an
+[RFC](https://github.com/void-linux/void-packages/issues/20935) from April
```

Comment:
First use of term should be defined; link format should be `void-linux/void-packages#20935`.

Reply:
The discussion started in an
    [Request For Comments \(RFC\)](void-linux/void-packages/issues/20935)

Reply:
@the-maldridge I'm not sure I understand what you mean by link format here. Something like `in an RFC, [void-linux/void-packages#20935](https://...)`?

Reply:
that's pretty much exactly what I mean.  the link should give you a good idea where it goes before you click on it.

Reply:
@ahesford "ongoing" gives me the impression that the decision is still in progress, so I won't use it for now.